uploadNeuroDB/tarchiveLoader

```diff
@@ -574,6 +574,17 @@ if (!$valid_study) {
     print "No Mincs inserted \n \n";
     exit 9;
 }
+
+###############################################################
+### Set Processed to 1 in mri_upload table#####################
+###############################################################
+my $where = "WHERE TarchiveID=?";
+my $query = "UPDATE mri_upload SET InsertionComplete=1 ";
```

Comment:
It can be done here at the same time as setting InsertionComplete = 1. I do it right at the return from this call (namely in the imaging_upload_file.pl; please see comment above), and that, irrespective of whether runTarchiveLoader() passed or failed.